source/css/_common/components/post/post-tags.styl

```diff
@@ -5,6 +5,5 @@
   a {
     display: inline-block;
     margin-right: 10px;
-    font-size: 13px;
```

Comment:
Screens: before and after?

Reply:
No changes are made here. `font-size` of `<a>` element is already defined with the same value. This line is no longer needed.